packages/integration-tests/src/test-project-setup/minimal_with_typescript_idioms.ts

```diff
@@ -66,10 +64,29 @@ class MinimalWithTypescriptIdiomTestProject extends TestProjectBase {
     super(name, projectDirPath, projectAmplifyDirPath, cfnClient);
   }
 
-  assertPostDeployment = async (): Promise<void> => {
-    const clientConfigStats = await fs.stat(
-      path.join(this.projectDirPath, 'amplifyconfiguration.json')
+  /**
+   * Creates a test project directory and instance.
+   */
+  static createProject = async (
+    e2eProjectDir: string,
+    cfnClient: CloudFormationClient
+  ): Promise<MinimalWithTypescriptIdiomTestProject> => {
+    const { projectName, projectRoot, projectAmplifyDir } =
```

Comment:
this looks like outcome of bad merge.
This got replaced by `MinimalWithTypescriptIdiomTestProjectCreator.createProject`

Reply:
Done in [8fa9444](https://github.com/aws-amplify/amplify-backend/pull/682/commits/8fa9444d2a1dd65c1f980e6ebbb143464eecdcd5)